integration-tests/kafka-sasl-ssl/src/main/resources/application.properties

```diff
@@ -0,0 +1,18 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+
```

Comment:
I wonder if it would be possible to configure this automatically from extension. For instance, based on the value of `camel.component.kafka.security-protocol`. This would be another ticket anyway, just thinking aloud.

Reply:
I have thought about that sort of thing in the past. I don't think it's always possible to resolve that configuration at build time. I.e when it's configured directly on the endpoint or via a component producer etc.

So we could probably make something work but maybe not for 100% of all configuration scenarios.

I also considered adding SSL support by default, assuming most folks will be taking the sensible route of securing their apps in production. Obviously that brings some additional overhead for those that don't need it though...

Reply:
yeah indeed, there are other cases at stake. And concerning the trade-off, I think it's fair enough right now. Let's see if we have feedback from the field about this.